src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html

```diff
@@ -52,6 +52,7 @@
           placeholder="500.00"
           value.bind="token.amount & validate"
           decimals.bind="token.decimals"
+          disabled.bind="token.decimals === undefined"
```

Comment:
I think we need a check for an empty value of the decimal input (in case the Address details doesn't contain the decimals value). When tested with the decimals input enabled (I just removed the "if.bind" statement from the input) and removed the decimals, an error showed up.

Reply:
Fixed by [947218d](https://github.com/PrimeDAO/prime-deals-dapp/pull/421/commits/947218d2c82e2223a6e75a4989bacea4945a2710)